distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -105,6 +105,11 @@ log4j2.logger.org_apache_cxf_jaxrs_impl_WebApplicationExceptionMapper.level = ER
 log4j2.logger.org_apache_cxf_phase_PhaseInterceptorChain.name = org.apache.cxf.phase.PhaseInterceptorChain
 log4j2.logger.org_apache_cxf_phase_PhaseInterceptorChain.level = ERROR
 
+# camel
+# Supresses an NPE warning that is ignored by Camel
+log4j2.logger.org_apache_camel_impl_defaultunitofwork.level = ERROR
```

Comment:
❓ How often is this exception happening?

Reply:
It occurs every time a file being monitored is deleted.